cni/network/network.go

```diff
@@ -153,12 +153,6 @@ func (plugin *NetPlugin) Start(config *common.PluginConfig) error {
 		zap.String("version", plugin.Version),
 		zap.String("component", "cni-net"))
 
-	log.Logger.Info("Os Info",
-		zap.String("platform", platform.GetOSInfo()),
-		zap.String("component", "cni-net"))
-	platform.PrintDependencyPackageDetails()
```

Comment:
We can remove the function `PrintDependencyPackageDetails` itself since this is the only place is being used.

Reply:
i wanted to keep for sometime incase if we need this to be logged